app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -0,0 +1,24 @@
+.header.mb-5
+  .std-heading
+    i.fa-solid.fa-wallet
+    =< t('.header')
+
+.container
+  .row
+    .col-8.col-md-4.col-lg-3
+      img.nbp_wallet_qr_code[src="#{nbp_wallet_qr_code_users_path}?truncated_reference=#{@truncated_reference}"]
```

Comment:
Ideally, we would still set some `alt` text for better accessibility (even if it is just explaining that only a QR code is displayed). This would, at least, satisfy the accessibility requirements public-funded projects have.

Reply:
Done